components/sections/Projects/ProjectItem.jsx

```diff
@@ -0,0 +1,35 @@
+import { useEffect, useRef, useState } from "react";
+import ProjectTitle from "./ProjectTitle";
+
+/* eslint-disable @next/next/no-img-element */
+const ProjectItem = ({ name, imgUri, description, projectUrl, setActive, active, index }) => {
+  const imgRef = useRef();
+  const [width, setWidth] = useState();
+
+  useEffect(() =>{
+    (() =>{
+      window && setWidth(window.innerWidth)
+    })()
+  }, []);
+
+  const openProjectPage = () => window.open(projectUrl, "blank");
+
+  const toggleImgState = () => imgRef.current.classList.toggle("project__image--active");
+
+  return (
+    <div className="flex flex-col xl:flex-row items-start project" onMouseEnter={() =>{setActive(index); toggleImgState()}} onMouseLeave={toggleImgState}>
```

Comment:
since `toggleImgState` is a one-line function.. we can use it inline 
```suggestion
    <div [..keep what was here.....] onMouseLeave={() => imgRef.current.classList.toggle("project__image--active")}>
```